src/services/resource.js

```diff
@@ -8,14 +8,17 @@
 const httpStatusCode = require('@generics/http-status')
 const resourceQueries = require('@database/queries/resources')
 const resourceCreatorMappingQueries = require('@database/queries/resourcesCreatorMapping')
+const reviewResourcesQueries = require('@database/queries/reviewsResources')
 const reviewsQueries = require('@database/queries/reviews')
+const reviewStagesQueries = require('@database/queries/reviewStage')
 const responses = require('@helpers/responses')
 const common = require('@constants/common')
 const userRequests = require('@requests/user')
+const configs = require('@services/config')
 const _ = require('lodash')
 const { Op } = require('sequelize')
-const axios = require('axios')
 const utils = require('@generics/utils')
+const axios = require('axios')
 const filesService = require('@services/files')
 
 module.exports = class resourceHelper {
```

Comment:
@adithyadinesh0412 Please include function signature for all functions in this file - name of function, description, use of each argument the data type, and sample values of each argument and also sample return values.

The list function is too complicated. It should be as simple as src/services/modules.js and other functions can call with various filters etc.

We should consider having page wise functions, each which is called based on page query in controller.